.github/workflows/build-deploy-api-v2.yaml

```diff
@@ -0,0 +1,119 @@
+name: Github Environment - DPIA API v2
+
+on:
+  push:
+    branches: [ main ]
+    paths:
+      - "src/backend/**"
+  workflow_dispatch:
+  
+jobs:
+  DPIA-API-Build-Tag-Push:
+    environment: API-Dev
+    runs-on: ubuntu-latest
+    steps:
+    
+      # Checkout CIRMO-DPIA Github Repository
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      # Login the BC Gov Openshift (DPIA TOOLS Namespace)
+      - name: Cluster Login -> TOOLS Namespace
+        uses: redhat-developer/openshift-actions@v1.1
+        with:
+          openshift_server_url: ${{ secrets.OPENSHIFT4_SERVER_URL }}
+          parameters: '{"apitoken": "${{ secrets.OPENSHIFT4_TOOLS_SA_PASSWORD }}"}'
+          cmd: |
+            'version'
+            
+      # Login to BC Gov Docker Image Repository
+      - name: Login to Openshift Docker
+        run : |
+          docker login ${{ secrets.OPENSHIFT4_PUBLIC_IMAGE_REPOSITORY_BASE_URL }} -u ${{ secrets.OPENSHIFT4_TOOLS_SA_USERNAME }} -p ${{ secrets.OPENSHIFT4_TOOLS_SA_PASSWORD }}
+          
+      # Setup environment variables
+      - name: Setup environment
+        run: |
+          make api-setup-env
+      
+      # Build the DPIA API Application Image via docker-compose
+      - name: Build the Docker image
+        run: |
+          docker-compose build dpia-api
+      
+      # TROUBLESHOOTING - view images present in github action run
+      - name: Docker List Images
+        run: |
+          docker image ls 
+      
+      # Tag Docker Image built with latest
+      - name: Docker Tag Version
+        run: |
+          docker tag cirmo-dpia_dpia-api ${{ secrets.OPENSHIFT4_API_PUBLIC_IMAGE_REPO_DEV }}:latest
+
+      # Push docker image to BC Gov Repository  
+      - name: Docker Push Version
+        run: |
+          docker push ${{ secrets.OPENSHIFT4_API_PUBLIC_IMAGE_REPO_DEV }}:latest
```

Comment:
This doesn't feel like a secret to me. I think it could just be a regular env var

Reply:
cool, working on it.

Reply:
I think I've resolved this.